### DISCUSSION TITLE
Possible values for engine options

### DISCUSSION THREAD

**Discussion by lotfire** [vpzomtrrfrt/stability-client#14] [Q&A] [answered]
Hello all,

first thanks for the work.
I wanted to test the different version from v1 to v2 so I tried to change the engine option but only "stable-diffusion-v1-5" is working.

All other attempts return:
`
  status: 'NotFound',
  code: 5,
  message: 'The specified engine (ID stable-diffusion-v1-4) was not found.',
`

Any help ?

cheers

**Comment by vpzomtrrfrt** [comment, marked as answer]
Based on [this PR](https://github.com/Stability-AI/stability-sdk/pull/147/files), the valid values appear to be `stable-diffusion-v1 stable-diffusion-v1-5 stable-diffusion-512-v2-0 stable-diffusion-768-v2-0 stable-inpainting-v1-0 stable-inpainting-512-v2-0`

**Comment by lotfire** [comment]
thanks @vpzomtrrfrt, it works !
